plugins/tiddlywiki/sqlite3store/init-sqlite3.js

```diff
@@ -0,0 +1,66 @@
+/*\
+title: $:/plugins/tiddlywiki/sqlite3store/init-sqlite3.js
+type: application/javascript
+
+Initialise sqlite3 and then boot TiddlyWiki
+
+This file is spliced into the HTML file to be executed after the boot kernel has been loaded.
+
+\*/
+
+(function() {
+
+// Get the main tiddler store out of the HTML file
+var storeEl = document.querySelector("script.tiddlywiki-tiddler-store"),
+	tiddlerStore = JSON.parse(storeEl.textContent);
+
+// Helper to get a tiddler from the store by title
+function getTiddler(title) {
+	for(var t=0; t<tiddlerStore.length; t++) {
+		var tiddler = tiddlerStore[t];
+		if(tiddler.title === title) {
+			return tiddler;
+		}
+	}
+	return undefined;
+}
+// Get the shadow tiddlers of this plugin
+var thisPlugin = getTiddler("$:/plugins/tiddlywiki/sqlite3store"),
+	thisPluginTiddlers = JSON.parse(thisPlugin.text).tiddlers;
+// Execute the sqlite3 module
+var sqlite3js = thisPluginTiddlers["$:/plugins/tiddlywiki/sqlite3store/sqlite3.js"].text,
+	context = {
+		exports: {}
+	};
+$tw.utils.evalSandboxed(sqlite3js,context,"$:/plugins/tiddlywiki/sqlite3store/sqlite3.js",true);
+// Create a Blob URL for the wasm data
+var sqlite3wasm = thisPluginTiddlers["$:/plugins/tiddlywiki/sqlite3store/sqlite3.wasm"].text;
```

Comment:
Why this, instead of `$tw.wiki.getTiddlerText` ?

Context is https://github.com/Jermolene/TiddlyWiki5/discussions/7947